.circleci/config.yml

```diff
@@ -1,33 +1,39 @@
 version: 2.1
 
-jobs:
-  package-macos:
-    parameters:
-      xcode-version:
-        type: "string"
+# Define executors that various packaging jobs need.  This lets us make one
+# packaging job that accepts an executor parameter and avoids duplicating the
+# packaging steps.
+executors:
+  manylinux-2014-x86_64:
+    docker:
+      - image: "quay.io/pypa/manylinux2014_x86_64"
+  manylinux_2_24-x86_64:
+    docker:
+      - image: "quay.io/pypa/manylinux_2_24_x86_64"
 
-    macos:
-      xcode: "<< parameters.xcode-version >>"
+  # Since this executor doesn't use a pypa-defined image created specifically
+  # for building wheels, we're a little bit less confident of exactly what
+  # kind of manylinux wheel will come out of it.  x_y here is a reference to
+  # the "future-proof" (heh) PEP 600 manylinux definition.  For a given pinned
+  # image, we should probably always get the same version out (eg
+  # manylinux_2_17 as of ubuntu-2004:202101-01) but if glibc on the system is
+  # upgraded then we'll probably get higher numbers.
+  manylinux_x_y-aarch64:
+    machine:
+      image: "ubuntu-2004:202101-01"
+    resource_class: "arm.large"
```

Comment:
Looking at https://github.com/pypa/manylinux#docker-images it appears there *are* manylinux aarch64 images for 2014 (i.e. 2.17?) and 2.24.

Reply:
Indeed - but we have to use a CircleCI-supplied "machine" image here because that's the only way to get an arm7 system from CircleCI (as far as I can tell).

Reply:
(I suppose I could set this up to run Docker on the CircleCI-supplied Ubuntu host but I'm not sure it's worth it just for more aarch64 wheels right now)

Reply:
(I guess I could explain some of that in the comment in the config file)

Reply:
That seems reasonable, but it definitely wasn't clear; I missed that this had `machine` as opposed to the `docker` above (which seems like it would be easy to do). Do we have any consumers of these wheels now even? 

I guess we don't have any dependencies other that glibc, so it shouldn't matter that we are using a non-manylinux image to build of somewhat random version with random installed software.

I think I'd *not* name the executor `manylinux-*`, given that it is an image with a [bunch](https://discuss.circleci.com/t/linux-machine-executor-images-2021-november-q4-update/42005) of software.

Reply:
Since we have no consumers of these wheels and because of the questions raised here, I filed #43 and removed the aarch64 executor and job.